Doc/library/math.rst

```diff
@@ -374,6 +376,24 @@ Floating point manipulation functions
    .. versionadded:: 3.2
 
 
+.. function:: isnormal(x)
+
+   Return ``True`` if *x* is a normal floating-point number, that is a finite
+   nonzero number that is not a subnormal (see :func:`issubnormal`).
+   Return ``False`` otherwise.
+
+   .. versionadded:: next
+
+
+.. function:: issubnormal(x)
+
+   Return ``True`` if *x* is a subnormal floating-point number, that is a finite
+   nonzero number with a magnitude smaller than the smallest positive normal
+   number, see :data:`sys.float_info.min`.  Return ``False`` otherwise.
```

Comment:
```suggestion
   Return ``True`` if *x* is a subnormal number, that is a finite
   nonzero number with a magnitude smaller than the smallest positive normal
   number, see :data:`sys.float_info.min`.  Return ``False`` otherwise.
```